application/src/main/java/com/sanction/thunder/authentication/UserAuthenticator.java

```diff
@@ -0,0 +1,46 @@
+package com.sanction.thunder.authentication;
+
+import com.google.common.base.Optional;
+import com.sanction.thunder.dao.PilotUsersDao;
+import com.sanction.thunder.models.PilotUser;
+import io.dropwizard.auth.Authenticator;
+import io.dropwizard.auth.basic.BasicCredentials;
+
+import javax.inject.Inject;
+
+public class UserAuthenticator implements Authenticator<BasicCredentials, User> {
+  private final PilotUsersDao usersDao;
+
+  @Inject
+  public UserAuthenticator(PilotUsersDao usersDao) {
+    this.usersDao = usersDao;
+  }
+
+  public PilotUsersDao getUsersDao() {
```

Comment:
We don't need this method. You can access the `usersDao` directly in this class, and no other class needs to get this property.